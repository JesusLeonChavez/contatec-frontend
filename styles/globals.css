:root{
  --primary: #706F6F;
  --secondary: #304659;
}

html,
body {
  padding: 0;
  margin: 0;

}

a {
  color: inherit;
  text-decoration: none;
}

* {
  box-sizing: border-box;
  font-family: 'Poppins', sans-serif;
}

.generalWrapper{
  max-width: 1300px;
  margin: 0 auto;
  overflow: auto;
  padding: 0 40px;
}

.flexCenterCustom{
  display: flex;
  align-items: center;
  justify-content: center;
}


/*----------------Swiper-----------------------*/
.swiper-container {
  width: 100%;
  height: 100%;
}

.swiper-slide {
  text-align: center;
  font-size: 18px;
  background: red;
  /* Center slide text vertically */
  display: -webkit-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  align-items: center;
}

.slide-2{
  background: green;
}

.mySwiper  .swiper-button-next, .mySwiper  .swiper-button-prev {
  color: white;
  border: 1px solid white;
  border-radius: 50%;
  padding: 1.3rem;
}

.mySwiper .swiper-button-next{
  top: 800px;
  right: 100px;
}

.mySwiper .swiper-button-prev{
  top: 800px;
  right: 180px;
  left: auto;
}

.mySwiper .swiper-button-next::after,.mySwiper .swiper-button-prev::after{
  font-size: 1rem;
}


.mySwiper2{
  width: 200px;
}
.mySwiper2 .swiper-container{
  width: 100%;
  height: 100%;
}
.mySwiper2 .swiper-slide{
  text-align: center;
  font-size: 18px;
  background: #fff;
  background: pink;
  /* Center slide text vertically */
  display: -webkit-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  align-items: center;
}
.mySwiper2 .slide-2{
  background: blue;
}


.mySwiper3{
  width:100%;
}
.mySwiper3 .swiper-container{
  width: 300px;
  height: 100%;
}
.mySwiper3 .swiper-slide{
  text-align: center;
  font-size: 18px;
  background: #fff;
  background: purple;
  height: 500px;
  /* Center slide text vertically */
  display: -webkit-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  align-items: center;
}
.mySwiper3 .slide-2{
  background: blue;
}

.mySwiper3 .swiper-button-next, .mySwiper3 .swiper-button-prev {
  color: white;
  border: 1px solid white;
  border-radius: 50%;
  padding: 1.3rem;
}

.mySwiper3 .swiper-button-next{
  top: 450px;
  right: 100px;
}
.mySwiper3 .swiper-button-prev{
  top: 450px;
  right: 180px;
  left: auto;
}

.mySwiper3 .swiper-button-next::after, .mySwiper3 .swiper-button-prev::after{
  font-size: 1rem;
}




