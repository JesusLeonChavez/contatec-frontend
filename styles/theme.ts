import { extendTheme } from '@chakra-ui/react'
import { ButtonStyles as Button } from './components/buttonStyles'
export const newTheme = extendTheme({
  colors: {
    primary: '#EB0029',
    secondary: '#FF6F91',
    highlight: '#00C9A7'
  },
  components: {
    Button
  }
})
