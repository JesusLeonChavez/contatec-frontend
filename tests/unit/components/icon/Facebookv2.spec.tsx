/**

 * @jest-environment jsdom

 */

 import { render } from "@testing-library/react"
 import "@testing-library/jest-dom"
 import { newTheme } from "../../../../styles/theme"
 import { ChakraProvider } from "@chakra-ui/react"
 import Facebookv2 from "../../../../components/Icon/Facebookv2"
 
 const ChakraRenderer = ({ children }) => {
   return <ChakraProvider theme={newTheme}>{children}</ChakraProvider>
 }
 
 describe("Facebookv2", function () {
   test("debe renderizarse el componente", () => {
     const component = render(<Facebookv2 />, {
       // eslint-disable-next-line @typescript-eslint/ban-ts-comment
       // @ts-ignore
       wrapper: ChakraRenderer
     })
 
     expect(component).toMatchSnapshot()
   })
 })
 
 
