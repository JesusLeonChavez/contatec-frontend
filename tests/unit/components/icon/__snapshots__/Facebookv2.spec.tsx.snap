// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Facebookv2 debe renderizarse el componente 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    class="chakra-ui-light"
  >
    <div
      id="chakra-toast-portal"
    >
      <ul
        id="chakra-toast-manager-top"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-top-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-top-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-bottom-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
    </div>
    <div>
      <svg
        fill="none"
        height="1.5em"
        viewBox="0 0 29 28"
        width="1.5em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M28.25 13.75C28.25 6.156 22.094 0 14.5 0a13.7 13.7 0 00-9.265 3.59C2.48 6.104 1.083 9.725 1.083 13.75c0 7.594 5.823 13.534 13.417 13.534 4.023 0 7.642-1.511 10.156-4.265a13.7 13.7 0 003.594-9.269z"
          fill="#fff"
        />
        <path
          d="M18.048 8.871h1.775a.444.444 0 00.443-.444v-2.66a.444.444 0 00-.443-.444H17.16a4.879 4.879 0 00-4.879 4.879v3.548H9.621l-.681 2.702 3.675.473.075 9.834 3.465.292.563-.502V17.3h2.315c.203 0 .38-.14.43-.337l.665-2.66a.444.444 0 00-.43-.552h-2.98v-3.548c0-.735.596-1.331 1.33-1.331z"
          fill="#1877F2"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      fill="none"
      height="1.5em"
      viewBox="0 0 29 28"
      width="1.5em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M28.25 13.75C28.25 6.156 22.094 0 14.5 0a13.7 13.7 0 00-9.265 3.59C2.48 6.104 1.083 9.725 1.083 13.75c0 7.594 5.823 13.534 13.417 13.534 4.023 0 7.642-1.511 10.156-4.265a13.7 13.7 0 003.594-9.269z"
        fill="#fff"
      />
      <path
        d="M18.048 8.871h1.775a.444.444 0 00.443-.444v-2.66a.444.444 0 00-.443-.444H17.16a4.879 4.879 0 00-4.879 4.879v3.548H9.621l-.681 2.702 3.675.473.075 9.834 3.465.292.563-.502V17.3h2.315c.203 0 .38-.14.43-.337l.665-2.66a.444.444 0 00-.43-.552h-2.98v-3.548c0-.735.596-1.331 1.33-1.331z"
        fill="#1877F2"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
